base_layer/wallet_ffi/Cargo.toml

```diff
@@ -7,13 +7,13 @@ version = "0.18.6"
 edition = "2018"
 
 [dependencies]
-tari_comms = { version = "^0.10", path = "../../comms" }
+tari_comms = { version = "^0.10", path = "../../comms", features = ["c_integration"]}
 tari_comms_dht = { version = "^0.10", path = "../../comms/dht", default-features = false }
 tari_common_types = {path="../common_types"}
 tari_crypto = { git = "https://github.com/tari-project/tari-crypto.git", branch = "main" }
 tari_key_manager = { version = "^0.10", path = "../key_manager" }
 tari_p2p = { version = "^0.10", path = "../p2p" }
-tari_wallet = { version = "^0.10", path = "../wallet", default-features = false }
+tari_wallet = { version = "^0.10", path = "../wallet", features = ["c_integration"]}
```

Comment:
Pretty sure that the c_integration flag does nothing in the wallet lib, the default-features = false should mean that this crate does not use the bundled sql lib when building wallet. Not using the bundled Sqlite should be an exception to the default.